paddle/fluid/framework/op_registry.cc

```diff
@@ -45,7 +49,8 @@ static VariableNameMap ConvertOpDescVarsToVarNameMap(
 }
 
 std::unique_ptr<OperatorBase> OpRegistry::CreateOp(
-    const proto::OpDesc& op_desc) {
+    const proto::OpDesc& op_desc, const std::vector<std::string>& input_names,
```

Comment:
same above, `op_desc` also hold the `input_names_` info

Reply:
`OpDesc` could be constructed without `input_names`, for example, in unit test.